lib/services.template.ejs

```diff
@@ -20,8 +20,7 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' &&
     var m = url.match(/^(?:https?:)?\/\/([^\/]+)/);
     return m ? m[1] : null;
   }
-
-  var urlBaseHost = getHost(urlBase) || location.host;
+  var urlBaseHost = getHost(urlBase) ? urlBase : location.host;
```

Comment:
Makes sense. It would be better to make such change in a standalone pull request, but I can live with everything in a single commit too.

One thing though - could you please add a code comment explaining why `urlBaseHost` is potentially a full path instead of a hostname only, and linking to #251?